js/src/common/utils/SubtreeRetainer.js

```diff
@@ -32,7 +32,7 @@ export default class SubtreeRetainer {
    * @return {Object|false}
    * @public
    */
-  retain() {
+  needsRebuild() {
```

Comment:
Please rename / change the docblock example accordingly.

Reply:
Also, we could implement this with `this.callbacks.some(...)` now. :grin: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some